src/devices/Qmc5883l/Registers.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Iot.Device.Qmc5883l
+{
+    /// <summary>
+    /// registeries used by QMC5883L
+    /// </summary>
+    internal enum Registers : byte
+    {
+        QMC_CONFIG_REG_1_ADDR = 0x09,
```

Comment:
And here, the naming are ok because it's internal register, from the documentation.